extensions/wikia/ImageReview/ImageReviewStatsCache.class.php

```diff
@@ -35,87 +27,92 @@ class ImageReviewStatsCache
 
 	/**
 	 * global registry object
-	 * @var $wg WikiaGlobalRegistry
+	 * @var $wg MemcachedPhpBagOStuff
 	 */
-	private $wg = null;
+	private $memc;
 
 	/**
 	 * ImageReviewStats constructor.
-	 * @param $user_id string
-	 * @param $wg WikiaGlobalRegistry
+	 * @param $userId string
 	 */
-	public function __construct( $user_id, $wg ) {
-		foreach ( $this->allowed_stats as $key ) {
+	public function __construct( $userId ) {
+		foreach ( $this->allowedStats as $key ) {
 			if ( $key == self::STATS_REVIEWER ) {
-				$cache_key = wfSharedMemcKey( 'ImageReviewSpecialController', 'v2', 'image_stats', $user_id, $key );
+				$cache_key = wfSharedMemcKey( 'ImageReviewSpecialController', 'v2', 'image_stats', $userId, $key );
 			} else {
 				$cache_key = wfSharedMemcKey( 'ImageReviewSpecialController', 'v2', 'image_stats', $key );
 			}
-			$this->image_stats_cache_keys[$key] = $cache_key;
+			$this->imageStatsCacheKeys[$key] = $cache_key;
 		}
-		$this->wg = $wg;
+		$this->memc = F::app()->wg->Memc;
 	}
 
 	/**
 	 * @return array
      */
 	public function getAllowedStats() {
-		return $this->allowed_stats;
+		return $this->allowedStats;
 	}
 
 	public function getStatsKey( $key ) {
-		if ( !in_array( $key, $this->allowed_stats ) ) {
+		if ( !in_array( $key, $this->allowedStats ) ) {
 			$this->warning( "ImageReviewLog", [
 				'method' => __METHOD__,
 				'message' => 'Incorrect cache stats key',
 				'stats_key' => $key,
 			] );
 			return '';
 		}
-		return $this->image_stats_cache_keys[$key];
+		return $this->imageStatsCacheKeys[$key];
 	}
 
 	public function getStats() {
 		$stats = [];
-		foreach( $this->image_stats_cache_keys as $key => $cache_key ) {
-			$value = $this->wg->memc->get( $cache_key );
+		foreach( $this->imageStatsCacheKeys as $key => $cacheKey ) {
+			$value = $this->memc->get( $cacheKey );
 			if ( !empty( $value ) ) {
 				$stats[$key] = $value;
 			}
 		}
 		return $stats;
 	}
-
-	public function setStats ( $new_stats ) {
-		foreach ( $new_stats as $key => $value ) {
-			$cache_key = $this->getStatsKey( $key );
-			if ( empty( $cache_key ) ) {
+	
+	public function setStats( $newStats ) {
+		foreach ( $newStats as $key => $value ) {
+			$cacheKey = $this->getStatsKey( $key );
+			if ( empty( $cacheKey ) ) {
 				continue;
 			}
 
-			$this->wg->memc->set( $cache_key, $value, self::CACHE_EXPIRE_TIME );
+			$this->memc->set( $cacheKey, $value, self::CACHE_EXPIRE_TIME );
 		}
 	}
 
+	public function clearStats() {
+		foreach( $this->imageStatsCacheKeys as $key => $cacheKey ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$key" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)